include/rdma/fabric.h

```diff
@@ -121,6 +121,7 @@ typedef struct fid *fid_t;
 #define FI_TRANSMIT_COMPLETE	(1ULL << 27)
 #define FI_DELIVERY_COMPLETE	(1ULL << 28)
 
+#define FI_SELECTIVE_COMPLETION	(1ULL << 55)
```

Comment:
> See this comment near the top of fabric.h:

Right, I see now what that comment was attempting to convey, but it didn't match up with how I was thinking about the flag space (my natural desire was to assign unique values for every single flag with no reuse as long as there were still bits available).

> If you're still in the code, can you fix the spelling at part of the same commit?

Yes, already done as a separate tiny commit that I'll push in a moment.  I was just waiting to see if any other comments came in before pushing.